src/amo/components/StaticPages/About/index.js

```diff
@@ -0,0 +1,140 @@
+              %(start_link)sbecome an Add-on Reviewer%(end_link)s ! Our reviewers are add-on fans 
+              with a technical background who review add-ons for code quality and stability.`),
+              {
+                start_link: '<a href="https://wiki.mozilla.org/Add-ons/Reviewers/Applying">',
+                end_link: '</a>',
+              }), ['a'])}
+            />
+            <li
+              dangerouslySetInnerHTML={sanitizeHTML(i18n.sprintf(i18n.gettext(`Help improve this website.
+              It's open source, and you can file bugs and submit patches.
+              %(github)s contains all of our current bugs, legacy bugs can still be found in Bugzilla.`),
+              {
+                github: '<a href="https://github.com/mozilla/olympia/issues">GitHub</a>',
+              }), ['a'])}
+            />
+          </ul>
+          <p>{i18n.gettext(`If you're interested in add-ons but not quite as technical, there are still ways to help:`)}</p>
+          <ul>
+            <li>{i18n.gettext('Tell your friends! Let people know which add-ons you use.')}</li>
+            <li
+              dangerouslySetInnerHTML={sanitizeHTML(i18n.sprintf(i18n.gettext(`Participate in our %(forumsLink)s.`),
+                {
+                  forumsLink: '<a href="https://discourse.mozilla-community.org/c/add-ons">forums</a>',
+                }), ['a'])
+              }
+            />
+            <li>
+              {i18n.gettext('Review add-ons on the site. Add-on authors are more likely to improve their add-ons and write new ones when they know people appreciate their work.')}
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('I have a question')}</h2>
+          <p>{i18n.gettext('A good place to start is our')}&nbsp;
+            <Link to="/faq"><abbr title="{i18n.gettext('Frequently Asked Questions')}">FAQ</abbr></Link>
+            {i18n.gettext(`. If you don't find an answer there, you can`)}&nbsp;<a href="https://discourse.mozilla-community.org/c/add-ons">{i18n.gettext('ask on our forums')}</a>.
+          </p>
+          <p
+            dangerouslySetInnerHTML={sanitizeHTML(i18n.sprintf(i18n.gettext(`If you really need to contact someone
+            from the Mozilla team, please see our %(start_link)scontact information%(end_link)s page.`),
+            {
+              start_link: '<a href="https://developer.mozilla.org/en-US/Add-ons#Contact_us">',
+              end_link: '</a>',
+            }), ['a'])}
+          />
+        </section>
+        <section>
+          <h2>{i18n.gettext('Who works on this website?')}</h2>
+          <p>{i18n.gettext('Over the years, many people have contributed to this website, including both volunteers from the community and a dedicated AMO team. A list of significant contributors can be found on our ')}&nbsp;
+            <Link to="/pages/credits">{i18n.gettext('Site Credits')}</Link>&nbsp;{i18n.gettext(' page')}.
```

Comment:
@tofumatt, is there some equivalent JS API we could use to derive links so that we don't need to split up strings that use `Link`?

Reply:
This isn't needed anymore since the FAQ page and credits pages are being dropped.